crates/nu-command/src/debug/metadata.rs

```diff
@@ -20,7 +20,10 @@ impl Command for Metadata {
 
     fn signature(&self) -> nu_protocol::Signature {
         Signature::build("metadata")
-            .input_output_types(vec![(Type::Nothing, Type::Record(vec![]))])
+            .input_output_types(vec![
+                (Type::Nothing, Type::Record(vec![])),
```

Comment:
You don't need `Type::Nothing` if you have `Type::Any`

Reply:
ooh that's a good point :ok_hand:

Reply:
should be fixed in 369342376